e2e/components/OverflowMenu/OverflowMenu-test.avt.e2e.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { expect, test } = require('@playwright/test');
+const { visitStory } = require('../../test-utils/storybook');
+
+test.describe('OverflowMenu @avt', () => {
+  test('accessibility-checker', async ({ page }) => {
+    await visitStory(page, {
+      component: 'OverflowMenu',
+      id: 'components-overflowmenu--default',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('OverflowMenu');
+  });
+
+  test('accessibility-checker render custom icon', async ({ page }) => {
+    await visitStory(page, {
+      component: 'OverflowMenu',
+      id: 'components-overflowmenu--render-custom-icon',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('OverflowMenu-custom-icon');
+  });
+
+  test('accessibility-checker open menu', async ({ page }) => {
+    await visitStory(page, {
+      component: 'OverflowMenu',
+      id: 'components-overflowmenu--default',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page.getByRole('button')).toBeVisible();
+
+    // Tab and open the toggle button for the OverflowMenu
+    await page.keyboard.press('Tab');
+    const toggleButton = page.getByRole('button');
+    await expect(toggleButton).toBeFocused();
+    await page.keyboard.press('Enter');
+    await expect(toggleButton).toHaveClass(/cds--overflow-menu--open/);
+
+    await expect(page).toHaveNoACViolations('OverflowMenu-open');
+  });
+
+  test('accessibility-checker render custom icon open', async ({ page }) => {
+    await visitStory(page, {
+      component: 'OverflowMenu',
+      id: 'components-overflowmenu--render-custom-icon',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page.getByRole('button')).toBeVisible();
+
+    // Tab and open the toggle button for the OverflowMenu
+    await page.keyboard.press('Tab');
+    const toggleButton = page.getByRole('button');
+    await expect(toggleButton).toBeFocused();
+    await page.keyboard.press('Enter');
+    await expect(toggleButton).toHaveClass(/cds--overflow-menu--open/);
```

Comment:
@tay1orjones do we want this to be `toBeVisible` instead of `toHaveClass`?

Reply:
I think you are right @andreancardona 
It might be better to check if the `ul` is visible instead of checking for the `className`, because the CSS can be edited. I'm going to change that!

Reply:
Yep that's what I was chatting with Taylor about - thanks :)